patches/api/0363-Expand-FallingBlock-API.patch

```diff
@@ -0,0 +1,66 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Owen1212055 <23108066+Owen1212055@users.noreply.github.com>
+Date: Sun, 5 Dec 2021 14:58:55 -0500
+Subject: [PATCH] Expand FallingBlock API
+
+- add auto expire setting
+- add setter for block data
+
+diff --git a/src/main/java/org/bukkit/World.java b/src/main/java/org/bukkit/World.java
+index da524a71af74b02515b037f7fe09ba6988e2c8bf..f82f24a15a74557c9a466949e12b2b67f7d2e11b 100644
+--- a/src/main/java/org/bukkit/World.java
++++ b/src/main/java/org/bukkit/World.java
+@@ -2257,8 +2257,10 @@ public interface World extends RegionAccessor, WorldInfo, PluginMessageRecipient
+      * @return The spawned {@link FallingBlock} instance
+      * @throws IllegalArgumentException if {@link Location} or {@link
+      *     BlockData} are null
++     * @deprecated use {@link #spawn(Location, Class, Consumer)} (or a variation thereof) in combination with {@link FallingBlock#setBlockData(BlockData)}
+      */
+     @NotNull
++    @org.jetbrains.annotations.ApiStatus.Obsolete // Paper
```

Comment:
There are 2 other `spawnFallingBlock` methods and although they are already deprecated, the deprecation notice should be updated to point to the new method. We like to avoid double jumps in deprecation notices, like one deprecated method that points to another that points to the actual one.